pages/pipelines/managing_log_output.md

```diff
@@ -209,4 +209,10 @@ To set up a private build log archive storage:
     }
     ```
 
+    >📘
+    > Folder structure and file format will be as follow: {BUILDKITE_PIPELINE_ID}/{BUILDKITE_BUILD_ID}/{BUILDKITE_JOB_ID}.log. This file format and folder structure is currently not customisable. 
```

Comment:
I think this also belongs in the intro section rather than during the procedure steps. I'll add a suggestion for how I see that working.

```suggestion
```